docs/WALKTHROUGH.md

```diff
@@ -80,6 +80,61 @@ export function handleDeposit(event: Deposit): void {
 
 Play around with this, compare hashes on etherscan and help yourself understand some of the behavior.
 
+## Using Logs in Subgraphs
+
+There is a standard logging format that we follow in subgraphs to ensure subgraph data consumers have a framework to understand what is happening in a subgraph. There are five different types of log severity levels that we use in subgraphs:
+
+- `log.debug`: This is only used for debugging purposes. It should not be found in any subgraph code in our `master` branch.
+- `log.info`: This is used for general information about the subgraph. It can be used to log handler execution, data processing, etc. This should never be used in the case of error handling.
+- `log.warning`: This is used for issues/warnings that may or may not be an issue. The issue will not create any major issues in a subgraph and should not affect downstream data consumers. This could be a divide by 0, but it may be the actual value so it is not an error that needs to be fixed.
+- `log.error`: This is used for errors that need to be addressed, but are not fatal to the subgraph's function. This should be cause for a fix, but the subgraph can continue to execute. This may cause a minor data issue that should be addressed soon. This would be synonymous to a [#minor](https://github.com/messari/subgraphs/blob/master/docs/CONTRIBUTING.md#minor) fix in our versioning system.
+- `log.critical`: This will be used for fatal errors. When this log is reached a subgraph will stop executing. This error will be cause for a major data issue that would create a major issue downstream if the data is consumed. This needs to be fixed ASAP and is synonymous to a [#major](https://github.com/messari/subgraphs/blob/master/docs/CONTRIBUTING.md#major) fix in our versioning system.
```

Comment:
The value passed to `log.critical` is not shown in the logs, it only shows that it stopped with a critical error.

We might require people to always log a `log.error` right before the `log.critical` so we can see what was the actual message

Reply:
I ran a test of this. It does actually show up. It just doesn't show up in the error message field when pinging for the health of a subgraph from the graph api.

See the pic in this msg: https://discord.com/channels/953684103012683796/1090341413700501524/1090833508664086578

Reply:
Oh! This is either new, or they don't show in the UI in the hosted-service (which is where I haven't seen them). But this works then!